src/components/Favorites.tsx

```diff
@@ -1,8 +1,44 @@
+import { useState, useEffect } from 'react';
+import api from '../services/api';
+import classNames from 'classnames';
+
+import ContactCard from './ContactCard';
+import DropDown from './DropDown';
+import View from './View';
+
+import '../styles/Favorites.scss';
+
 const Favorites = () => {
+    const [favoriteContacts, setFavoriteContacts] = useState<any[] | null>(null);
+    const [sortValue, setSortValue] = useState<number | null>(null);
+    const [view, setView] = useState('grid');
+
+    useEffect(() => {
+        api.getAllFavorites().then(res => setFavoriteContacts(res));
+    }, [])
+
+    const trigger = (e: any) => {
```

Comment:
Even if you decide to use event object, please provide the interface here instead of any